modules/refined/src/test/scala/doobie/refined/RefinedSuite.scala

```diff
@@ -54,15 +52,15 @@ class RefinedSuite extends munit.FunSuite {
   }
 
   test("Query should return a refined type when conversion is possible") {
-    sql"select 123".query[PositiveInt].unique.transact(xa).void.unsafeRunSync()
+    sql"select 123".query[PositiveInt].unique.transact(xa).void.assert
   }
 
   test("Query should return an Option of a refined type when query returns null-value") {
-    sql"select NULL".query[Option[PositiveInt]].unique.transact(xa).void.unsafeRunSync()
+    sql"select NULL".query[Option[PositiveInt]].unique.transact(xa).void.assert
```

Comment:
The same concern here:
```suggestion
    sql"select NULL".query[Option[PositiveInt]].unique.transact(xa)
      .assertEquals(None)
```

Reply:
Sounds good. Added this check.